EIPS/eip-5606.md

```diff
@@ -0,0 +1,114 @@
+     */
+    function bundle(DelegateData[] memory delegateData, uint256 proxyTokenID, address ownerAddress) external;
+
+    /**
+     * @dev Initializes a new bundle, mints a Proxy NFT and assigns it to msg.sender
+     * Returns the token ID of a new Proxy NFT
+     * Note - When a new Proxy NFT is initialized, it is empty, it does not contain any delegate NFTs
+     */
+    function initBundle(DelegateData[] memory delegateData) external;
+}
+```
+
+Any dapp implementing this standard would initialise a bundle by calling the function `initBundle`. This mints a new proxy NFT and assigns it to msg.sender. While creating a bundle, the delegate token contract addresses and the token IDs are set during the initialisation and cannot be changed after that. This avoids unintended edge cases where non-related NFTs could be bundled together by mistake.
+
+Once a bundle is initialised, the delegate NFT tokens can then be transferred to this Proxy NFT contract by calling the function `bundle` and passing the token ID of the proxy NFT. It is important for a dapp to get the delegate NFTs ‘approved’ from the owner to this Proxy NFT contract before calling the bundle function. After that, the Proxy NFT owns one or more versions of this digital asset across the various platforms.
+
+If the owner of the proxy NFT wants to sell or use the individual delegate NFTs across any of the platforms, they can do so by calling the function `unbundle`. This function transfers the particular delegate NFT token(s) to msg.sender (only if msg.sender is the owner of the proxy NFT).
+
+## Rationale
+
+Since their inception, NFTs were meant to be interoperable and prove the scarcity of digital assets. Although NFTs can arguably prove the scarcity of items, the interoperability aspect hasn’t been addressed yet. Creating a proxy NFT standard that allows for indexing and ownership of a digital asset across various platforms would be the first step towards interoperability and true ownership in the multiverse.
+
+In the web3 ecosystem, NFTs have evolved to represent multiple types of unique and non-fungible assets. One type of asset includes a set of NFTs related to one another. For instance, if a brand releases a new sneaker across various metaverses, it would be minted as a separate NFT on each platform. However, it is, in reality, the same sneaker across metaverses.
+
+There is a need to represent the relationship and transferability of these types of NFTs as metaverses and blockchain games gain more mainstream adoption. The ecosystem needs a better framework to address this issue rather than relying on the application level. This framework should define the relationship between these assets and the nature of their association. There is more value in the combined recognition, use and transferability of these individual NFTs as a bundle/proxy rather than their selves.
```

Comment:
All of this is more Motivation than Rationale. The Rationale section should describe the reasoning behind individual technical choices within this EIP, while the Motivation explains why the EIP as a whole should exist.

Reply:
A good example for this section would be to explain why `bundle`, and `unbundle` take an array of `DelegateData`.